reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,294 @@
+.. index::
+   single: Forms; Fields; dateinterval
+
+dateinterval Field Type
+=======================
+
+This field type allows the user to modify data that represents a specific
+interval (e.g. ``P7Y6M5DT12H15M30S``).
```

Comment:
Displaying this strange interval string at the beginning of the article seems a bit odd to me. What about rewording the first two sentences as follows?

```
This field type allows the user to modify data that represents a specific
date interval. It can be rendered as a integer or text input or select tags.

The underlying format of the data can be a ``DateInterval`` object,
a `RFC 3339`_ duration string (e.g. ``P7Y6M5DT12H15M30S``) or an array.
```